tests/module/module.info.yml

```diff
@@ -0,0 +1,6 @@
+name: Module
+type: module
+description: Drupal module with all sorts of tests, to test capabilities of the drupal-testing package.
+package: Drupal Testing
+core: 8.x
+core_version_requirement: ^8 || ^9
```

Comment:
Test modules don't need a core or core_version_requirement key

Reply:
Why not?

Reply:
Ah, I thought it's a real test module, but it's the module we want to test ;)